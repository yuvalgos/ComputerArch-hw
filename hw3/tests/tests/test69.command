./dflow_calc tests/test69.dat tests/test69.in p108 p97 d36 d92 p114 d182 d58 d14 p170 d193 p78 p41 d39 d91 d126 p60 p102 p189 p88 p22 p136 p154 p152 d28 p8 d83 d65 p172 d18 p180 d7 d130 d105 p171 p36 d50 p91 d188 p161 p5 p150 d97 d169 d132 p44 d145 d40 d187 p79 p10 d99 p123 d9 d124 d81 d138 p141 d25 p119 d189 d31 p147 p154 d37 d7 d173 d23 p80 p69 p78 d18 p48 p99 d178 d53 d195 p86 d182 d117 d154 p24 d174 d153 d69 d41 p43 d89 p35 d173 p157 d175 d19 p11 d43 d188 p40 p176 p115 d64 d59 d181 p38 p80 p92 p163 d94 p180