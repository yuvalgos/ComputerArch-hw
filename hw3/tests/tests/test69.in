1 11 16 28
0 11 28 16
4 17 31 6
8 11 30 21
7 7 2 23
0 16 1 9
1 7 22 14
0 22 13 18
5 3 21 6
6 29 4 24
4 24 20 31
7 12 11 0
3 30 13 15
7 0 1 28
8 0 30 21
4 15 3 19
3 1 4 7
1 6 16 17
6 8 10 7
7 25 0 31
7 19 29 16
5 19 6 29
1 5 30 20
4 26 13 11
1 31 21 28
6 0 3 20
0 22 12 9
7 5 6 15
3 30 9 5
2 11 30 15
3 30 1 3
7 20 21 9
8 30 23 27
6 12 30 21
4 10 6 10
5 1 26 22
8 16 15 31
0 6 5 18
3 13 15 13
2 0 14 25
3 1 18 27
4 18 22 24
1 3 13 21
6 12 19 27
3 10 5 4
2 20 4 21
6 27 16 23
5 16 14 30
4 24 11 4
3 21 24 14
3 17 20 9
2 19 9 25
6 8 13 31
2 11 21 4
1 12 6 14
3 3 4 29
1 27 26 23
2 11 0 31
1 8 11 13
6 30 23 13
1 5 3 13
2 31 28 7
7 7 26 17
2 2 8 20
8 5 28 15
7 25 14 1
0 27 24 0
6 27 10 0
0 14 26 21
6 15 18 20
5 1 0 11
8 10 22 20
6 26 28 20
6 29 12 20
0 11 27 5
5 0 3 14
4 7 15 28
0 21 11 30
5 20 6 14
6 14 10 6
8 17 0 5
3 29 4 20
2 30 16 1
1 2 4 18
3 4 21 17
6 18 25 22
1 0 31 25
6 14 1 18
4 0 3 14
5 26 5 27
5 11 3 0
8 8 4 1
3 29 6 19
3 24 13 13
3 31 9 22
1 16 16 9
5 22 13 9
3 30 31 31
5 23 22 31
6 16 18 2
5 20 27 5
8 23 25 27
8 7 7 23
6 7 29 20
1 14 2 11
5 17 8 11
4 16 14 29
4 30 7 15
0 6 27 2
2 16 7 0
2 28 15 8
3 28 0 17
8 20 14 9
8 24 6 24
6 11 11 18
1 7 7 9
8 0 16 14
3 3 5 15
1 0 16 5
5 17 1 4
7 20 9 27
1 9 11 11
6 3 9 17
7 24 5 25
6 20 2 20
8 23 22 28
7 13 16 7
8 9 31 17
0 15 23 6
1 22 24 8
4 27 12 12
0 0 2 5
1 22 12 17
6 6 17 15
5 14 13 14
7 1 16 28
5 11 0 17
6 21 13 24
1 30 17 20
4 15 17 27
2 29 3 19
0 1 22 12
7 30 17 29
2 13 25 21
7 27 23 15
2 8 3 3
0 24 24 17
5 24 22 22
8 29 8 31
6 8 10 12
8 19 10 24
6 21 12 3
6 13 31 4
3 29 17 0
6 20 3 28
2 3 15 18
2 13 6 11
5 18 15 11
6 10 13 22
5 6 15 31
3 30 24 27
8 27 18 20
2 5 22 7
5 13 17 10
4 30 30 9
4 26 19 12
0 12 3 4
2 3 5 18
0 29 20 15
5 0 6 5
6 2 30 28
3 1 12 19
2 11 7 2
2 11 25 15
8 0 13 7
6 19 26 28
5 15 1 2
7 29 20 22
3 22 24 4
7 0 3 21
7 19 4 25
3 19 8 29
1 27 14 29
3 21 16 5
2 3 16 14
7 6 20 27
5 27 16 6
2 30 16 24
3 17 12 31
2 8 30 8
3 14 19 0
8 11 22 8
5 10 16 11
1 2 19 29
7 16 5 15
1 21 28 4
8 15 14 9
0 30 31 0
7 2 0 6
0 19 8 25
