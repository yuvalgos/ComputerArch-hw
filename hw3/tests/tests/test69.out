Reading the opcodes latency info from tests/test69.dat ... Got latency for 9 opcodes
Reading the program file tests/test69.in ... Found 200 instructions
getProgDepth()==270
getDepDepth(108)==71
getDepDepth(97)==51
getInstDeps(36)=={15,24}
getInstDeps(92)=={37,51}
getDepDepth(114)==113
getInstDeps(182)=={134,177}
getInstDeps(58)=={57,38}
getInstDeps(14)=={12,-1}
getDepDepth(170)==221
getInstDeps(193)=={181,177}
getDepDepth(78)==75
getDepDepth(41)==44
getInstDeps(39)=={-1,19}
getInstDeps(91)=={84,70}
getInstDeps(126)=={109,115}
getDepDepth(60)==31
getDepDepth(102)==94
getDepDepth(189)==152
getDepDepth(88)==89
getDepDepth(22)==7
getDepDepth(136)==131
getDepDepth(154)==161
getDepDepth(152)==118
getInstDeps(28)=={-1,27}
getDepDepth(8)==0
getInstDeps(83)=={-1,41}
getInstDeps(65)=={-1,40}
getDepDepth(172)==238
getInstDeps(18)=={-1,6}
getDepDepth(180)==118
getInstDeps(7)=={-1,-1}
getInstDeps(130)=={54,54}
getInstDeps(105)=={91,90}
getDepDepth(171)==170
getDepDepth(36)==24
getInstDeps(50)=={45,-1}
getDepDepth(91)==118
getInstDeps(188)=={166,94}
getDepDepth(161)==178
getDepDepth(5)==0
getDepDepth(150)==147
getInstDeps(97)=={94,94}
getInstDeps(169)=={159,162}
getInstDeps(132)=={54,119}
getDepDepth(44)==43
getInstDeps(145)=={122,122}
getInstDeps(40)=={-1,-1}
getInstDeps(187)=={109,147}
getDepDepth(79)==67
getDepDepth(10)==0
getInstDeps(99)=={85,83}
getDepDepth(123)==47
getInstDeps(9)=={-1,-1}
getInstDeps(124)=={83,120}
getInstDeps(81)=={-1,78}
getInstDeps(138)=={119,124}
getDepDepth(141)==134
getInstDeps(25)=={8,-1}
getDepDepth(119)==118
getInstDeps(189)=={187,149}
getInstDeps(31)=={-1,-1}
getDepDepth(147)==134
getDepDepth(154)==161
getInstDeps(37)=={27,-1}
getInstDeps(7)=={-1,-1}
getInstDeps(173)=={65,139}
getInstDeps(23)=={-1,3}
getDepDepth(80)==88
getDepDepth(69)==55
getDepDepth(78)==75
getInstDeps(18)=={-1,6}
getDepDepth(48)==55
getDepDepth(99)==61
getInstDeps(178)=={147,84}
getInstDeps(53)=={49,-1}
getInstDeps(195)=={111,84}
getDepDepth(86)==26
getInstDeps(182)=={134,177}
getInstDeps(117)=={64,69}
getInstDeps(154)=={122,111}
getDepDepth(24)==0
getInstDeps(174)=={163,115}
getInstDeps(153)=={119,131}
getInstDeps(69)=={41,45}
getInstDeps(41)=={26,10}
getDepDepth(43)==30
getInstDeps(89)=={64,67}
getDepDepth(35)==44
getInstDeps(173)=={65,139}
getDepDepth(157)==144
getInstDeps(175)=={165,111}
getInstDeps(19)=={14,-1}
getDepDepth(11)==23
getInstDeps(43)=={21,-1}
getInstDeps(188)=={166,94}
getDepDepth(40)==0
getDepDepth(176)==238
getDepDepth(115)==101
getInstDeps(64)=={-1,15}
getInstDeps(59)=={-1,38}
getInstDeps(181)=={149,177}
getDepDepth(38)==24
getDepDepth(80)==88
getDepDepth(92)==54
getDepDepth(163)==225
getInstDeps(94)=={-1,26}
getDepDepth(180)==118
